README.md

```diff
@@ -12,6 +12,12 @@ EVE supports both ARM and Intel architectures and requires hardware-assisted vir
 
 To get its job done, EVE leverages a lot of great open source projects: [Xen Project](https://xenproject.org/), [Linuxkit](https://github.com/linuxkit/linuxkit) and [Alpine Linux](https://alpinelinux.org/) just to name a few. All of that functionality is being orchestrated by the Go microservices available under [pkg/pillar](pkg/pillar). Why pillar? Well, because pillar is the kind of a monolith we need to break out into true, individual microservices under [pkg/](pkg/).
 
+## Download EVE
+
+EVE LTS: [8.12.0-LTS](https://github.com/lf-edge/eve/releases/tag/8.12.0)
+
+EVE Latest: [9.2.0](https://github.com/lf-edge/eve/releases/tag/9.2.0)
```

Comment:
Would be nice if we can figure out to auto-update at least this latest, and ideally LTS as well. Can we create some links inside https://github.com/lf-edge/eve/releases/tag or https://github.com/lf-edge/eve/releases so we don't need up manually do a commit to README.md every two weeks.

Reply:
Good idea, let me check.